src/python/pants/backend/python/rules/pytest_runner.py

```diff
@@ -89,9 +92,9 @@ async def setup_pytest_for_target(
         python_setup,
     )
 
-    # Ensure all pexes we merge via PEX_PATH to form the test runner use the interpreter constraints
-    # of the tests. This is handled by CreatePexFromTargetClosure, but we must pass this through for
-    # CreatePex requests.
+    # Ensure all pexes we merge via --pex-path to form the test runner use the interpreter
+    # constraints of the tests. This is handled by PexFromTargetsRequest, but we must pass
+    # this through for PexRequest.
     pex_request = functools.partial(PexRequest, interpreter_constraints=interpreter_constraints)
```

Comment:
I'm not certain this still pulls its weight. It means that MyPy no longer understands the code and it requires 3 lines of comments to explain.

Fine to keep, though.